srcs/juloo.keyboard2/LauncherActivity.java

```diff
@@ -26,6 +29,39 @@ public void onCreate(Bundle savedInstanceState)
     if (VERSION.SDK_INT > 28)
       _tryhere_area.addOnUnhandledKeyEventListener(
           this.new Tryhere_OnUnhandledKeyEventListener());
+
+  }
+
+  private static final int REQUEST_CODE_OVERLAY_PERMISSION = 1601;
+  private void requestOverlayPermission() {
+    // Request the permission using an Intent
+    Intent intent = new Intent(Settings.ACTION_MANAGE_OVERLAY_PERMISSION, Uri.parse("package:" + getPackageName()));
```

Comment:
`Uri.parse` can be replaced by `Uri.fromParts("package", getPackageName(), "")`.